WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginEmailPasswordFragment.java

```diff
@@ -87,11 +84,38 @@ public void onCreate(Bundle savedInstanceState) {
         mService = getArguments().getString(ARG_SOCIAL_SERVICE);
         isSocialLogin = getArguments().getBoolean(ARG_SOCIAL_LOGIN);
 
-        if (savedInstanceState != null) {
+        if (savedInstanceState == null) {
+            // cleanup the service state on first appearance
+            LoginWpcomService.clearLoginServiceState();
+        } else {
             mRequestedPassword = savedInstanceState.getString(KEY_REQUESTED_PASSWORD);
         }
     }
 
+    @Override
+    public void onResume() {
+        super.onResume();
+
+        // connect to the Service. We'll receive updates via EventBus.
+        mServiceEventConnection = new AutoForeground.ServiceEventConnection(getContext(), LoginWpcomService.class, this);
+
+        // install the change listener as late as possible so the UI can be setup before triggering the
```

Comment:
the sentence in the comment is ended abruptly "before triggering the".

Reply:
:man_facepalming: what a cliffhanger!
Fixed with c8287d3.